charts/xp-treatment/Chart.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
   repository: https://caraml-dev.github.io/helm-charts
   version: 0.2.8
 description: Treatment service - A part of XP system that is used to obtain the treatment configuration from active experiments
-version: 0.1.3
-appVersion: 0.11.1
 maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
+name: xp-treatment
```

Comment:
Some reordering to standardize with all other Chart.yaml files in the repository.